conf/messages

```diff
@@ -392,6 +392,14 @@ authorisations.addAuthorisationsYesNo.checkYourAnswersLabel = Do you want to add
 authorisations.addAuthorisationsYesNo.change.hidden = if you want to add an authorisation
 authorisations.addAuthorisationsYesNo.error.required = Select yes if you want to add an authorisation
 
+
+authorisations.addArrivalDateYesNo.title = Do you want to add the arrival date at the office of destination?
```

Comment:
Again, very picky but could we amend the message keys to be in the form of `authorisationsAndLimit.limit.addLimitDateYesNo` to match the package structure?